src/components/ListingCard/ListingCard.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { UUID } from 'crypto';
+import { CaseListing } from '../../types/schemaTypes';
+import timestampStringToDate from '../../utils/helpers';
+import { CardBody, CardTitle, TagRow, CardTag } from './styles';
+
+export default function ListingCard({
+  caseData,
+  isSelected = false,
+  onClick,
+}: {
+  caseData: CaseListing;
+  isSelected?: boolean;
+  onClick?: (id: UUID) => void;
+}) {
+  // setup
+  const rolesNeeded = ['Attorney'].concat(
+    caseData.needs_interpreter ? ['Interpreter'] : [],
+  );
+
+  // helper functions
+  const parseDate = (d: Date): string =>
+    `${d.getMonth() + 1}/${d.getDate()}/${d.getFullYear()}`;
+
+  // temporary solution
+  const attorneyColor = '#85b4d2';
```

Comment:
Thoughts on moving this out to `colors.ts`? As in https://github.com/calblueprint/girls-write-now/blob/main/src/styles/colors.ts, for example

Reply:
Y'all could also look into using design tokens to define colors and such (https://www.smashingmagazine.com/2021/07/global-local-styling-nextjs/)